### PR TITLE
limit log size of logDispatcher

### PR DESCRIPTION
**Pull request by mychaow**

### CODE REVIEW

File: cluster/src/main/java/org/apache/iotdb/cluster/log/LogDispatcher.java [hunk 1]
```diff
@@ -289,32 +294,44 @@ private AppendEntriesRequest prepareRequest(List<ByteBuffer> logList,
...
-      for (SendLogRequest request : currBatch) {
-        Timer.Statistic.LOG_DISPATCHER_LOG_IN_QUEUE
-            .calOperationCostTimeFromStart(request.getLog().getCreateTime());
-        logList.add(request.getAppendEntryRequest().entry);
-      }
+      int logIndex = 0;
+      logger.info("send logs from index {} to {}", currBatch.get(0).getLog().getCurrLogIndex(),
+        currBatch.get(currBatch.size() - 1).getLog().getCurrLogIndex());
+      while (logIndex < currBatch.size()) {
+        long logSize = IoTDBDescriptor.getInstance().getConfig().getThriftMaxFrameSize();
```
Comment (neuyilan) [inline cluster/src/main/java/org/apache/iotdb/cluster/log/LogDispatcher.java:311]: maybe we can move the line outside of the while loop

**Review by neuyilan — COMMENTED** [review]
Great!

File: cluster/src/main/java/org/apache/iotdb/cluster/server/member/RaftMember.java [hunk 2]
```diff
@@ -1524,7 +1529,7 @@ private AppendLogResult sendLogToFollowers(Log log, AtomicInteger voteCounter) {
     try {
       if (allNodes.size() > 2) {
         // if there are more than one followers, send the requests in parallel so that one slow
-        // follower will not
+        // follower will not blocked
```
Comment (jt2594838) [inline cluster/src/main/java/org/apache/iotdb/cluster/server/member/RaftMember.java:1532]: be blocked

**Review by jt2594838 — APPROVED** [review]

**Comment by jixuan1989** [comment]
Any description (of the issue and the desgin) for this PR?

**Comment by mychaow** [comment]
> Any description (of the issue and the design) for this PR?

just a simple bug fix, so I don't create issue for it.

**Pull request merged**